### ISSUE TITLE
Typo on Input component's htmlType as hymlType, causing TS compiler to fail.

### ISSUE THREAD

**Issue by sannajammeh** [geist-org/react#598]
# Bug report 🐞

## Version & Environment

  - Version of browser Chrome 90
  - Version of `geist-ui/react` ^2.2.0-rc.8

## Expected Behaviour

The Input's attribute htmlType should not be hymlType. Cannot compile if I still wish to have password on my inputs.

## Actual results (or Errors)

I got an error:

```ts
// TS is happy with this
 <Input icon={<FiLock />} placeholder="p" hymlType="password" scale={1.5} />
// Not this
 <Input icon={<FiLock />} placeholder="p" htmlType="password" scale={1.5} />
```

**Comment by sannajammeh** [comment]
Temporary fix until merge.
```ts
// geist.d.ts or any declarations file in your project

declare module '@geist-ui/react/dist/input/input-props' {
    export interface Props {
        htmlType?: string;
    }
}
```

**Comment by ofekashery** [comment]
Fixed in #599.
